### DISCUSSION TITLE
Mindmap with Latex - Export to PDF corrupt formulas

### DISCUSSION THREAD

**Discussion by afriesinger** [freeplane/freeplane#1912] [Help]
I created a mind map with a lot of formulas. Written in Latex. in Freeplane everything is nicely shown. 
When i export the mind map to pdf. All the symbols are replaced by random signs and letters. Even normal Latex text like the word "fluid" in the example, isn't exported correctly. 
For me it looks like that the latex font isn't applied correctly during export. 
![grafik](https://github.com/freeplane/freeplane/assets/76171462/1ae4e65f-d6cb-4914-a72d-d2c2030c5e6d)

**Comment by edbeee** [comment]
I'm facing the similar issue.
Pdf screenshot:
![image](https://github.com/user-attachments/assets/7c0342ce-90f6-44bd-8d1a-12d2fa934df7)


freeplane screenshot:
![image](https://github.com/user-attachments/assets/6ca70e97-4d11-4b55-b0a1-caeddefcfec7)


when exported to png:
![image](https://github.com/user-attachments/assets/6bb974ac-2fa3-4b5f-919f-372916f20e1c)
